llvm/utils/release/build_llvm_release.bat

```diff
@@ -165,6 +167,24 @@ set common_cmake_flags=^
   -DLLVM_ENABLE_RPMALLOC=ON ^
   -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra;lld;compiler-rt;lldb;openmp"
 
+if "%force-msvc%" == "" (
+  where /q clang-cl
+  if errorlevel 0 (

```

Comment:
I suppose I should have caught this in review, but I ran into it when building 20.1.0-rc1 instead :-)

Apparently `if errorlevel 0` will succeed if the error level is `>= 0`, i.e. in our case we will take the branch also when `where /q clang-cl` is *not* successful. (E.g. https://devblogs.microsoft.com/oldnewthing/20080926-00/?p=20743)

I'll work around this by passing `--force-msvc`, but it also needs to be fixed.